README.md

```diff
@@ -468,6 +468,8 @@ end
 
 ### Schema Definition
 
+As noted above, you can use the examples below, or switch to the idiom in Rails 3.2+ which uses `def up` and `def down` instead of `def self.up` and `def self.down`, respectively.
+
```

Comment:
Rails <3.2 is end-of-lifed, and not supported any longer (https://github.com/thoughtbot/paperclip/blob/master/.travis.yml#L14). I would remove this comment and old snippets.